EE.C

```diff
@@ -0,0 +1,478 @@
+#define EE_cxx
+#include "EE.h"
+#include <TH2.h>
+#include <TStyle.h>
+#include <TCanvas.h>
+
+void EE::Loop()
+{
+//   In a ROOT session, you can do:
+//      root> .L EE.C
+//      root> EE t
+//      root> t.GetEntry(12); // Fill t data members with entry number 12
+//      root> t.Show();       // Show values of entry 12
+//      root> t.Show(16);     // Read and show values of entry 16
+//      root> t.Loop();       // Loop on all entries
+//
+
+//     This is the loop skeleton where:
+//    jentry is the global entry number in the chain
+//    ientry is the entry number in the current Tree
+//  Note that the argument to GetEntry must be:
+//    jentry for TChain::GetEntry
+//    ientry for DYree::GetEntry and TBranch::GetEntry
+//
+//       To read only selected branches, Insert statements like:
+// METHOD1:
+//    fChain->SetBranchStatus("*",0);  // disable all branches
+//    fChain->SetBranchStatus("branchname",1);  // activate branchname
+// METHOD2: replace line
+//    fChain->GetEntry(jentry);       //read all branches
+//by  b_branchname->GetEntry(ientry); //read only this branch
+   if (fChain == 0) return;
+
+   TFile *output = new TFile("DY_EE.root","recreate");
+
+   TH1F *run_DY = new TH1F("run_DY","run",3,0,3);
```

Comment:
You can and should always add axis descriptions to any histogram you declare. The fastest way to do this is in this example:
TH1F *run_DY = new TH1F("run_DY","run;x-axis description;y-axis description",3,0,3);

Reply:
For all histograms you should always call the Sumw2() method to get proper 1/sqrt(weights) uncertainties per bin. So in this example a second line should be added:
run_DY->Sumw2();